clustered/client/src/main/java/org/ehcache/clustered/client/internal/store/SimpleClusterTierClientEntity.java

```diff
@@ -288,4 +300,26 @@ private Duration getTimeoutDuration(EhcacheOperationMessage message) {
       }
     }
   }
+
+  private static void waitForLatch(long nanos, CountDownLatch latch) {
+    boolean interrupted = false;
+    long deadlineTimeout = System.nanoTime() + nanos;
+    try {
+      while (true) {
+        try {
+          long timeRemaining = deadlineTimeout - System.nanoTime();
+          if (timeRemaining <= 0) {
```

Comment:
After a timeout, instead of throwing an error, it is better to attempt to look at the response listeners. It is highly unlikely that init will not complete for such a large duration.